scripts/plotting/cloud_regime_analysis.py

```diff
@@ -0,0 +1,1369 @@
+#%%
+print()
```

Comment:
I would delete this line, as I suspect this `print` statement will be called at a weird time in the ADF run, if it is called at all.